src/main/cliapp/src/containers/conditionRelationPage/__tests__/ConditionRelationTable.test.js

```diff
@@ -18,19 +19,32 @@ describe("<ConditionRelationPage />", () => {
 	});
 
 	it("Renders without crashing", async () => {
-		const result = renderWithClient(<ConditionRelationPage />);
-		expect(result);
+		var result;
+		act(() => {
```

Comment:
These `act()` functions get rid of that warning when running the test which is awesome. Only thing I would say is change is change the `var` to `let.`

Reply:
I'll change them in next pr